lib/src/crop.dart

```diff
@@ -448,7 +453,7 @@ class CropState extends State<Crop> with TickerProviderStateMixin, Drag {
       }
     }
 
-    if (_action == _CropAction.cropping) {
+    if (_action == _CropAction.cropping && widget.canResizeGrid) {
```

Comment:
I would suggest adding this flag check in a condition above. Where `_action` gets assigned whether scaling, moving, or cropping. Idea is to avoid assigning cropping in first place if this flag is set to `false`.